app/controllers/api/v2/domains_controller.rb

```diff
@@ -33,6 +33,7 @@ def index
 
       api :GET, "/domains/:id/", N_("Show a domain")
       param :id, :identifier, :required => true, :desc => N_("Numerical ID or domain name")
+      param :show_hidden_parameter, :bool, :desc => N_("Should unhide hidden parameters values for display")
```

Comment:
I think the wording is wrong, both the "should" and the "unhide", I'd say "Display hidden parameters", "Reveal hidden parameters" or "Expose hidden parameters" convey the right meaning better.

Mind @domcleal @GregSutcliffe or some other native speaker to check it out? 

@orrabin Since tests pass already and the issue is just with the strings, I'll likely change it on merge as it's just a string.